justfile

```diff
@@ -1,5 +1,5 @@
 # THIS FILE IS AUTOGENERATED FROM FLAKEBOX CONFIGURATION
-!include justfile.fedimint.just
+import "justfile.fedimint.just"
```

Comment:
this looks like its an issue with the version of flakebox? @dpc

Reply:
Yes, some older version of `just` used a different syntax.